sema/commons/j2/syntax_builder.py

```
@@ -66,13 +66,21 @@ def variables_in_template(self, name: str) -> set:  # type: ignore
+    ) -> str:  # type: ignore
         """
         Fills a named template sparql
 
         :param name: of the template
         :param **variables: named context parameters to apply to the template
         """
-        log.debug(f"building sparql query '{name}' with variables={variables}")
-        qry = self._get_rdfsyntax_template(name).render(variables)
+        log.debug(
+            f"building sparql query '{_template_name}' "
+            f"with variables={variables}"
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)